pcweb/pages/docs/getting_started/configuration.py

```diff
@@ -0,0 +1,100 @@
+import pynecone as pc
+
+from pcweb.templates.docpage import (
+    doccode,
+    docheader,
+    doclink,
+    docpage,
+    doctext,
+    subheader,
+docalert
+)
+
+CONFIG_REF_URL = "/docs/api-reference/config"
+
+config_example1 = """
+# pcconfig.py
+import pynecone as pc
+class ExpConfig(pc.Config):
+    pass
+    
+config = ExpConfig(
+    env_path = "path/to/env/file",
+    ...
+)
+
+"""
+
+config_example2 = """
+# pcconfig.py
+import pynecone as pc
+class ExpConfig(pc.Config):
+    env_path = "path/to/env/file"
+
+config = ExpConfig(
+    ...
+)
+
+"""
+
+@docpage()
+def configuration():
+    return pc.box(
+        docheader("Configuration"),
+        doctext("There are 3 ways to configure your pynecone application."),
+        doctext(""),
+        subheader("PCConfig Arguments"),
+        doctext(
+            "The first place to configure your pynecone application is ",
+            "by setting the different options in the",
+            pc.code("pcconfig.py"),
+            " file.",
+        ),
+        pc.text(
+            "Refer to the ",
+            doclink("Config API Reference ", href=CONFIG_REF_URL),
+            "for the details of all options availables."
+        ),
+        subheader("Environment Arguments"),
+        doctext(
+            "By default, pynecone looks for a .env file (with ",
```

Comment:
I think this part is repetitive - can we just say looks for a `.env` file in your root directory